README.md

```diff
@@ -190,7 +190,34 @@ gnag {
 ***NOTE:*** All reporters are enabled by default
 
 - ***enabled*** - easily disable Gnag in specific situations
-- ***failOnError*** - should violations cause the build to fail or just generate a report
+- ***failOnError*** - should violations cause the build to fail or just generate a report; if set to false, you may need to add the following to prevent your build still failing from Android Lint errors:
+
+    <details open>
+    <summary><b>groovy</b></summary>
+
+    ```
+    android {
+        lintOptions {
+            abortOnError false
+        }
+    }
+    ```
+    
+    </details>
+    
+    <details open>
```

Comment:
Also to remain consistent with the rest of the Kotlin docs remove the `open` option here and have it just be `<details>`